calendar_event_jitsi/data/mail_template_data.xml

```diff
@@ -0,0 +1,249 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
```

Comment:
Correct me if i'm wrong, but aren't these templates not going to be installed?
They're set as noupdate=1 in the core module: https://github.com/odoo/odoo/blob/13.0/addons/calendar/data/mail_data.xml#L3

so how would it work?

Reply:
@ivantodorovich .. all 3 mail template updated with our custom code while we install "calendar_event_jitsi" module first time in any database.

So there is no issue with "noupdate='1'" tag

Reply:
Hmmm ok, I'm curious though.. how come they're installed?
I see they're set as noupdate=True in core.

Reply:
yes, i know that they are set as noupdate=True. but if you install new module then it will update view however base view has noupdate =True.

Reply:
That's good to know, has it always been like this or is this new in 13.0?

Reply:
always